### ISSUE TITLE
what will happen if the input of torch.nn.functional.binary_cross_entropy is zero

### ISSUE THREAD

**Issue by hanhaowen** [pytorch/pytorch#50658]
## ❓ Questions and Help

If  the input tensot of torch.nn.functional.binary_cross_entropy is zero , and the target tensor is 1.0, the output will be 100. 
Here is my code:

input=torch.tensor(0.0)
target=torch.tensor(1.0)
loss=F.binary_cross_entropy(input,target)
print(loss)  #tensor(100.)

so I want to know how does the function proceed when the input is zero, cause in the math formula, log(0.0) will cause a math domain problem , but this does not happen in the pytorch.

I am looking forward to hearing from you 

cc @jlin27 @mruberry @albanD @jbschlosser

**Comment by ngimel** [comment]
"100" value is hardcoded here https://github.com/pytorch/pytorch/blob/7e05d07ca7557b0b433b97a5943c8eedbbf34837/aten/src/ATen/native/Loss.cpp#L146-L148. Leaving open to document this behavior.

**Comment by mruberry** [comment]
We would accept a PR clarifying the documentation.

**Comment by thomasjpfan** [comment]
This behavior currently documented in `BCELoss`:

https://github.com/pytorch/pytorch/blob/59b2b8b091727f496cd3136eb1fd7004d8f7748b/torch/nn/modules/loss.py#L566-L568

Is this enough for documentation purposes?
